BibBuddy/app/src/main/java/de/bibbuddy/BookNotesView.java

```diff
@@ -279,42 +283,64 @@ private void setupAddButton() {
     PopupMenu pm = new PopupMenu(getContext(), addButtonView);
     pm.getMenuInflater().inflate(R.menu.add_note_menu, pm.getMenu());
 
-    pm.setOnMenuItemClickListener(new PopupMenu.OnMenuItemClickListener() {
-      @Override
-      public boolean onMenuItemClick(MenuItem item) {
-        if (item.getItemId() == R.id.add_text_note) {
-          Bundle bundle = new Bundle();
-          bundle.putLong(LibraryKeys.BOOK_ID, bookId);
-          TextNoteEditorFragment nextFrag = new TextNoteEditorFragment();
-          nextFrag.setArguments(bundle);
-          getActivity().getSupportFragmentManager().beginTransaction()
-              .replace(R.id.fragment_container_view, nextFrag,
-                  LibraryKeys.FRAGMENT_TEXT_NOTE_EDITOR)
-              .addToBackStack(null)
-              .commit();
-
-        } /* else if (item.getItemId() == R.id.add_voice_note) {
-                    TODO: add features to add voice notes
-                }*/
-
-
-        return true;
+    pm.setOnMenuItemClickListener(item -> {
+      if (item.getItemId() == R.id.add_text_note) {
+        Bundle bundle = new Bundle();
+        bundle.putLong(LibraryKeys.BOOK_ID, bookId);
+        TextNoteEditorFragment textFrag = new TextNoteEditorFragment();
+        textFrag.setArguments(bundle);
+        requireActivity().getSupportFragmentManager().beginTransaction()
+            .replace(R.id.fragment_container_view, textFrag,
+                LibraryKeys.FRAGMENT_TEXT_NOTE_EDITOR)
+            .addToBackStack(null)
+            .commit();
```

Comment:
A few new lines for better readability :)

```suggestion
    pm.setOnMenuItemClickListener(item -> {
    
      if (item.getItemId() == R.id.add_text_note) {
        Bundle bundle = new Bundle();
        bundle.putLong(LibraryKeys.BOOK_ID, bookId);
        TextNoteEditorFragment textFrag = new TextNoteEditorFragment();
        textFrag.setArguments(bundle);
        
        requireActivity().getSupportFragmentManager().beginTransaction()
            .replace(R.id.fragment_container_view, textFrag,
                LibraryKeys.FRAGMENT_TEXT_NOTE_EDITOR)
            .addToBackStack(null)
            .commit();
            
```